boilerplate/app/app.tsx

```diff
@@ -9,6 +10,9 @@
  * The app navigation resides in ./app/navigators, so head over there
  * if you're interested in adding screens and navigators.
  */
+if (__DEV__) {
+  require("./services/reactotron")
```

Comment:
❤️

Reply:
Push up a commit with a comment for this:
https://github.com/infinitered/ignite/pull/2465/commits/1037d3ce290debf7e98cdbb203b443f51efe77c8